spec/spec_helper.rb

```diff
@@ -31,6 +31,21 @@ def travis?
   # https://relishapp.com/rspec/rspec-core/docs/command-line/only-failures
   config.example_status_persistence_file_path = 'tmp/example_status.txt'
 
+  config.before do |scenario|
+    selenium_ver = ENV["BUNDLER_ORIG_BUNDLE_GEMFILE"]
```

Comment:
If you really want selenium-webdriver version, you can get it by

```ruby
Selenium::WebDriver::VERSION
```

and you can get its major version by

```ruby
selenium_major_ver = Gem::Version.new(Selenium::WebDriver::VERSION).segments[0]
```

or

```ruby
selenium_major_ver = Selenium::WebDriver::VERSION.split('.')[0]
```